src/main/NWNX/API/Object.cs

```diff
@@ -7,54 +7,121 @@ namespace NWNX.API
 {
   public static class Object
   {
-    static Object()
-    {
+    static Object() => 
       PluginUtils.AssertPluginExists<ObjectPlugin>();
-    }
+    
+    public static int GetLocalVariableCount(this NwObject obj) => 
+      ObjectPlugin.GetLocalVariableCount(obj);
 
-    public static void AddToArea(this NwGameObject gameObject, Location location)
-    {
+    public static LocalVariable GetLocalVariable(this NwObject obj, int index) => 
+      ObjectPlugin.GetLocalVariable(obj, index);
+
+    public static NwObject StringToObject(string id) =>
+      ObjectPlugin.StringToObject(id).ToNwObject();
+
+    public static void SetPosition(this NwObject obj, Location position, bool updateSubareas = true) =>
+      ObjectPlugin.SetPosition(obj, position.Position, updateSubareas ? 1 : 0);
```

Comment:
I don't think there's a use case where updateSubareas will ever need to be false (In NWNX, this was kept just to be safe).

This can be removed as it is implemented in `NwGameObject.Position`.